jina/parsers/hubble/new.py

```diff
@@ -0,0 +1,70 @@
+"""Argparser module for hub new"""
+
+import argparse
+
+from ..helper import add_arg_group
+
+
+def mixin_hub_new_parser(parser):
+    """Add the arguments for hub new to the parser
+    :param parser: the parser configure
+    """
+    gp = add_arg_group(parser, title='New')
+    gp.add_argument(
+        '-n',

```

Comment:
can you remove the single letter alias, we don't have such convention for `argsparse`. Reason is tho it may be convenient, but because we have so many args in our CLI, it can easily introduce conflict & hard to manage